runners/reference/java/build.gradle

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply from: project(":").file("build_rules.gradle")
+applyJavaNature()
+
+description = "Apache Beam :: Runners :: Reference :: Java"
+
+dependencies {
+  shadow project(path: ":beam-model-parent:beam-model-pipeline", configuration: "shadow")
+  shadow project(path: ":beam-runners-parent:beam-runners-core-construction-java", configuration: "shadow")
+  shadow library.java.slf4j_api
+  testCompile library.java.junit
+  testCompile library.java.slf4j_jdk14
+  testCompile library.java.hamcrest
```

Comment:
library.java.hamcrest_core?

Applying build_rules.gradle to beam-runners-reference-java
applyJavaNature with default configuration for project beam-runners-reference-java

FAILURE: Build failed with an exception.

* Where:
Build file '/home/jenkins/jenkins-slave/workspace/beam_PreCommit_Java_GradleBuild/src/runners/reference/java/build.gradle' line: 30

* What went wrong:
A problem occurred evaluating project ':beam-runners-parent:beam-runners-reference-parent:beam-runners-reference-java'.
> Cannot convert a null value to an object of type Dependency.
  The following types/formats are supported:
    - Instances of Dependency.
    - String or CharSequence values, for example 'org.gradle:gradle-core:1.0'.
    - Maps, for example [group: 'org.gradle', name: 'gradle-core', version: '1.0'].
    - FileCollections, for example files('some.jar', 'someOther.jar').
    - Projects, for example project(':some:project:path').
    - ClassPathNotation, for example gradleApi().

Reply:
Fixed